### DISCUSSION TITLE
Creating a link in Markdown within the same repo

### DISCUSSION THREAD

**Discussion by lagbolt** [community/community#23148] [New to GitHub]
<p>In Markdown is there a simple way to create a link to a file within the same repo?</p>
<p>That is, other than giving the full URL to the file.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You can use relative URLs, e.g. if you have two files:</p>
<pre><code class="lang-nohighlight">README.md
abc/test.md
</code></pre>
<p>In <code>README.md</code> you could create a link like this:</p>
<pre><code class="lang-auto">Check my [test document](./abc/test.md)!
</code></pre>

**Comment by lagbolt** [comment]
<p>Thanks!  That worked!  (Well, I guess you knew it worked, but it was news to me, because it’s not strictly the URL, which includes the branch name and “blob”.)</p>
